java-racingcar/src/test/java/com/game/javaracingcar/level1/SetTest.java

```diff
@@ -0,0 +1,48 @@
+package com.game.javaracingcar.level1;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.apache.logging.log4j.util.Strings;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class SetTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @DisplayName("요구사항 1")
+    @Test
+    void Requirements01 () {
+        assertThat(numbers.size()).isEqualTo(3);
+    }
+
+    @DisplayName("요구사항 2")
+    @ParameterizedTest
+    @ValueSource(ints = {1, 2, 3})
+    void Requirements02 (int input) {
+        assertThat(numbers.contains(input)).isTrue();
+    }
+
+    @DisplayName("요구사항 3")
+    @ParameterizedTest
+    @CsvSource(value = {"1:true", "2:true", "3:true", "4:false", "5:false"}, delimiter = ':')
+    void Requirements03 (int input, boolean expected) {
```

Comment:
안 써도 public인 거는 아는데 명시해 주는 게 가시적으로 좀 더 좋을 것 같아요.
```suggestion
    public void Requirements03 (int input, boolean expected) {
```